helpo-api/task-definition.json

```diff
@@ -0,0 +1,123 @@
+{
+    "ipcMode": null,
+    "executionRoleArn": "arn:aws:iam::838345163227:role/ecsTaskExecutionRole",
+    "containerDefinitions": [
+      {
+        "dnsSearchDomains": null,
+        "logConfiguration": {
+          "logDriver": "awslogs",
+          "secretOptions": null,
+          "options": {
+            "awslogs-group": "/ecs/helpo-api-task-definition",
+            "awslogs-region": "us-west-2",
+            "awslogs-stream-prefix": "ecs"
+          }
+        },
+        "entryPoint": [],
+        "portMappings": [
+          {
+            "hostPort": 80,
```

Comment:
Vamos a tener un ALB in front of este container escuchando el 443 y sirviendo certificados y demás boludeces para HTTPS TSL/SSL/ETC/CTO/TKM verdad?

Reply:
Sí, entiendo que si pero estaba probando primero como anda la Action. Hasta mergearlo no puedo verlo bien